setup.py

```diff
@@ -63,7 +63,8 @@
 DLL_HASH_ENV = "BEZIER_DLL_HASH"
 REQUIREMENTS = ("numpy >= 1.18.1",)
 EXTRAS_REQUIRE = {
-    "full": ["matplotlib >= 3.0.0", "scipy >= 1.4.1", "sympy >= 1.5.1"]
+    "full": ["matplotlib >= 3.0.0", "scipy >= 1.4.1", "sympy >= 1.5.1"],
+    ':python_version<"3.8"': ["importlib-metadata"],
```

Comment:
I should probably also limit this to just Windows.

Reply:
Probably should add a reference to [PEP 508][1] and/or [PEP 496][2], I always forget what they say.

The check `platform_system == "Windows"` "works best" here (or `os_name == "nt"`).

[1]: https://www.python.org/dev/peps/pep-0508/
[2]: https://www.python.org/dev/peps/pep-0496/